src/ontology/mondo-edit.obo

```diff
@@ -19704,6 +19704,7 @@ subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "Choroiditides" EXACT [MESH:D002833]
 synonym: "choroiditis" EXACT [NCIT:C35111, Orphanet:280892]
+synonym: "Posterior Uveitis" EXACT [NORD:1601]
 synonym: "Posterior Uveitis" RELATED [NORD:1601]
```

Comment:
The new NORD content has "Posterior Uveitis" as an exact synonym so the `RELATED` will be removed.
<img width="668" alt="Screenshot 2024-08-22 at 2 35 59 AM" src="https://github.com/user-attachments/assets/e4ed39da-043e-4cfd-8c4b-a8fdc9c9d5f1">

Reply:
This should have been caught by the Mondo Ingest QC system, so we need to see why it has not!

Reply:
In mondo, `update-nord` only handles xref and subset annotations, e.g. `grep -vE '^(xref: NORD:|subset: nord_rare)' ...` I gather once the synonym sync is in place this will address cases where synonym scopes are different. 

In mondo-ingest, the `robot_report_external_content.txt` file does not yet include this qc sparql query, but I will add it. 

Does that cover what to check?